src/Microsoft.CodeQuality.Analyzers.Exp/Core/FlowAnalysis/Analysis/NullAnalysis/NullAnalysis.NullDataFlowOperationWalker.cs

```diff
@@ -0,0 +1,319 @@
+            {
+                var unused = base.VisitInvocation_NonLambdaOrDelegateOrLocalFunction(operation, argument);
+                return GetValueBasedOnInstanceOrReferenceValue(operation.Instance);
+            }
+
+            public override NullAbstractValue VisitFieldReference(IFieldReferenceOperation operation, object argument)
+            {
+                var unused = base.VisitFieldReference(operation, argument);
+                return GetValueBasedOnInstanceOrReferenceValue(operation.Instance);
+            }
+
+            public override NullAbstractValue VisitPropertyReference(IPropertyReferenceOperation operation, object argument)
+            {
+                var unused = base.VisitPropertyReference(operation, argument);
+                return GetValueBasedOnInstanceOrReferenceValue(operation.Instance);
+            }
+
+            public override NullAbstractValue VisitDynamicMemberReference(IDynamicMemberReferenceOperation operation, object argument)
+            {
+                var unused = base.VisitDynamicMemberReference(operation, argument);
+                return GetValueBasedOnInstanceOrReferenceValue(operation.Instance);
+            }
+
+            public override NullAbstractValue VisitMethodReference(IMethodReferenceOperation operation, object argument)
+            {
+                var unused = base.VisitMethodReference(operation, argument);
+                return GetValueBasedOnInstanceOrReferenceValue(operation.Instance);
+            }
+
+            public override NullAbstractValue VisitEventReference(IEventReferenceOperation operation, object argument)
+            {
+                var unused = base.VisitEventReference(operation, argument);
+                return GetValueBasedOnInstanceOrReferenceValue(operation.Instance);
+            }
+
+            public override NullAbstractValue VisitArrayElementReference(IArrayElementReferenceOperation operation, object argument)
+            {
+                var unused = base.VisitArrayElementReference(operation, argument);
+                return GetValueBasedOnInstanceOrReferenceValue(operation.ArrayReference);
+            }
+
+            public override NullAbstractValue VisitDynamicInvocation(IDynamicInvocationOperation operation, object argument)
+            {
+                var unused = base.VisitDynamicInvocation(operation, argument);
+                return GetValueBasedOnInstanceOrReferenceValue(operation.Operation);
+            }
+
+            public override NullAbstractValue VisitDynamicIndexerAccess(IDynamicIndexerAccessOperation operation, object argument)
+            {
+                var unused = base.VisitDynamicIndexerAccess(operation, argument);
+                return GetValueBasedOnInstanceOrReferenceValue(operation.Operation);
+            }
+
+            public override NullAbstractValue VisitAddressOf(IAddressOfOperation operation, object argument)
```

Comment:
Why is this isolated from all of the other methods that don't look like copy&paste?